src/extension.ts

```diff
@@ -308,16 +298,6 @@ export function activate(context: VSCode.ExtensionContext) {
     })
   );
 
-  context.subscriptions.push(
-    VSCode.commands.registerCommand(Commands.UPDATE_SERVERS_AND_BINDING_STORAGE, () => {
-      updateServerStorage()
-        .then(updateProjectBinding)
-        .then(() => {
-          VSCode.window.showInformationMessage('SonarLint server storage updated');
```

Comment:
If I understand correctly, users will now have to look at their LS logs to get feedback. I think we should try to find a way to provide more high level feedback about synchronization progress.

Reply:
We can show a message from the LS to the UI at the end of the process. I will have a look.